<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalStandardLoadEditDialog</class>
 <widget class="QDialog" name="CalStandardLoadEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Load Standard</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="name"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="coefficients">
       <property name="text">
        <string>Coefficients</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="measurement">
       <property name="text">
        <string>Measurement file</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_29">
           <property name="text">
            <string>Resistance:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="SIUnitEdit" name="resistance"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_30">
           <property name="text">
            <string>Z0:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="SIUnitEdit" name="Z0">
           <property name="enabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Offset delay [ps]:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="SIUnitEdit" name="delay"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Offset loss [GÎ©/s]: </string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="SIUnitEdit" name="loss"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>Parallel C:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="SIUnitEdit" name="parC"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_26">
           <property name="text">
            <string>Series L:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="SIUnitEdit" name="serL"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Load Parameter Model</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <item>
           <widget class="QRadioButton" name="L_first">
            <property name="text">
             <string>Series L first</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup_2</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="C_first">
            <property name="text">
             <string>Shunt C first</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup_2</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1">
       <item>
        <widget class="QLabel" name="measurementLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="updateFile">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Update from file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="TouchstoneImport" name="touchstoneImport" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SIUnitEdit</class>
   <extends>QLineEdit</extends>
   <header>CustomWidgets/siunitedit.h</header>
  </customwidget>
  <customwidget>
   <class>TouchstoneImport</class>
   <extends>QWidget</extends>
   <header>CustomWidgets/touchstoneimport.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CalStandardLoadEditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CalStandardLoadEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
